addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18880,7 +18880,19 @@ msgctxt "#36618"
 msgid "Add-ons will be given access to personal data stored on this device. By allowing, you agree that you are solely responsible for any loss of data, unwanted behaviour, or damage to your device. Proceed?"
 msgstr ""
 
-#empty strings from id 36619 to 36899
+#. Label of a setting, allow the user to specifiy high quality downscaling of pictures
+#: system/settings/settings.xml
+msgctxt "#36619"
+msgid "High quality downscaling"
+msgstr ""
+
+#. Description of setting "Pictures -> Slideshow -> High Quality Downscaling" with label #36619
+#: system/settings/settings.xml
+msgctxt "#36620"
+msgid "Enable high quality downscaling of pictures."
```

Comment:
> explain everything in the settings description like you would to a little child

but exactly this is not the case here. I would consider me a bad daddy, if I explained it like this to my daughter. she would like to know: "what does performance mean in this case? this is just a picture and it does not matter how long it takes to scale"